dbms/src/Storages/MergeTree/MergeTreeData.h

```diff
@@ -720,6 +721,17 @@ class MergeTreeData : public IStorage
 
     TTLEntry ttl_table_entry;
 
+    struct MoveTTLEntry
```

Comment:
Seems like `TTLEntry` is the same with `MoveTTLEntry`, but always with `DELETE` destination type. Isn't it?  I think we shouldn't make separate struct and use common TTLEntry.

Reply:
Fixed.